spatial-tracking-explainer.md

```diff
@@ -92,15 +92,15 @@ function onSessionStarted(session) {
 There is no mechanism for getting a floor-relative _unbounded_ reference space. This is because the user may move through a variety of elevations (via stairs, hills, etc), making identification of a single floor plane impossible.
 
 ### Stationary experiences
```

Comment:
Should we refer to these now as "local experiences" in the explainer, to avoid the same "stationary" ambiguity that we're avoiding in the API?